src/main/java/org/tron/common/runtime/Runtime.java

```diff
@@ -576,13 +576,15 @@ public void go() throws OutOfSlotTimeException {
         }
 
       } else {
-          deposit.commit();
+        deposit.commit();
       }
     } catch (OutOfResourceException e) {
       logger.error(e.getMessage());
       throw new OutOfSlotTimeException(e.getMessage());
     } catch (Exception e) {
       logger.error(e.getMessage());
+      runtimeError = e.getMessage();
+      e.printStackTrace();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Use a logger to log this exception. ([squid:S1148](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1148))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1287&issues=AWUisWcizFyjFVlutiQf&open=AWUisWcizFyjFVlutiQf)

[ISSUE_KEY]: AWUisWcizFyjFVlutiQf